Update documentation with new Submit script feature (#1357)
elyra-ai/elyra: docs/source/user_guide/enhanced-python-support.md

![Enhanced Python Support](../images/python-editor.gif)

The execution of the Python script leverages the available `Python based Kernels`. This enables
users to run their scripts in different configurations and environments.

> Elyra also allows submitting a Python script as a single node pipeline for execution in a Kubeflow or Airflow environment in the cloud. This feature is accessible when the Elyra [AI Pipelines](../user_guide/pipelines.md) extension is also enabled.

Reviewer: ```suggestion
Elyra also allows submitting a Python script as a single node pipeline for execution in a Kubeflow Pipelines or Apache Airflow environment in the cloud. This feature is accessible when the Elyra [AI Pipelines](../user_guide/pipelines.md) extension is also enabled.
```

## Python script execution support

* In the Jupyter Lab Launcher, click the `Python File` icon to create a new Python Script.
* When used in conjunction with `Jupyter Enterprise Gateway`, the dropdown will be populated with more kernel options,
allowing users to run their scripts with remote kernels with more specialized resources.
* To run your script locally, select the `python3` option in the dropdown menu, and click the `Run` icon.

## Ability to run a Python script as a batch job

> Elyra allows the execution of a Python script as a batch job in remote cloud environments. This feature leverages the AI pipelines feature and requires access to either a Kubeflow Pipelines or Airflow deployment via a [runtime configuration](../user_guide/runtime-conf).

Reviewer: ```suggestion
Elyra allows the execution of a Python script as a batch job in remote cloud environments. This feature leverages the AI pipelines feature and requires access to either a Kubeflow Pipelines or Apache Airflow deployment via a [runtime configuration](../user_guide/runtime-conf).
```

To submit a Python script as a batch job, open a Python script file and select `Submit Script...` button from the editor toolbar.

![Submit Python Script as a batch job](../images/submit-script.gif)

> To learn more about pipeline [runtimes configuration]((../user_guide/runtime-conf)) and [runtime images]((../user_guide/runtime-image-conf)), follow those sections provided in the Elyra user guide documentation.

Reviewer: ```suggestion
To learn more about [runtime configurations](../user_guide/runtime-conf) and [runtime images](../user_guide/runtime-image-conf), follow those sections provided in the Elyra user guide documentation.
```
